content/blog/2019/05/2019-05-30-becoming-contributor-newbie-tickets.adoc

```diff
@@ -0,0 +1,254 @@
+---
+layout: post
+title: "The journey of becoming a Jenkins contributor: Newbie-friendly tickets"
```

Comment:
Shorter titles often present better for readers.  Since "becoming" hints that the action is happening now, I think the reader will infer that this is describing a journey.
```suggestion
title: "Becoming a Jenkins contributor: Newbie-friendly tickets"
```